spec/features/fill_in_additional_information_page_spec.rb

```diff
@@ -19,15 +19,21 @@
     expect(confirmation_of_supplied_details_page).to be_displayed
   end
 
-  scenario "incorrectly will provide errors" do
-    pending("while we wait for a test environment to understand how the s3 bucket timings will work")
-    fail
-  end
-
   scenario "without uploading a file will not display 'Remove file' link on CoSD page" do
     additional_information_page.load
     additional_information_page.next
 
     expect(confirmation_of_supplied_details_page.confirmation_of_additional_information_answers).not_to have_link(t('components.confirmation_of_supplied_details.remove_file_link'), href: remove_rtf_path)
   end
+
+  scenario "correctly will leave a file in the s3 bucket" do
+    additional_information_page.load
+
+    given_i_am(:company01)
```

Comment:
Should this not be first ?  Its the 'setup' phase of the test isnt it ?

Reply:
As discussed - this is the way I've done it in the other specs. Happy to change if it ever becomes an issue